src/Potentials.jl

```diff
@@ -6,6 +6,11 @@ module Potentials
 using Unitful
```

Comment:
@yangmr04, I think clear instructions for implementing a new interatomic potential would be helpful to include in the developer guide.

Reply:
Good idea! Aligns with the concept of "building an interface".